docs/installation.rst

```diff
@@ -122,6 +122,22 @@ After all dependencies have been installed, we can build the code:
         cmake ..
         make
 
+Configuring the dynamics
+------------------------
+The dynamics for nextSIM are chosen at the point of configuring CMake. This is in contrast to most of the model configuration, which is done at model run time. The dynamics are set through the configuration option ``DynamicsType``. The available options for the dynamics are
```

Comment:
Should probably put nextSIM-DG.

Reply:
No. The general plan is to refer to what we call nextSIM-DG  as plain nextSIM in the future. Plus, it is used without the DG elsewhere in the same document.